app/src/main/java/com/yeonae/chamelezone/data/source/remote/member/MemberRemoteDataSourceImpl.kt

```diff
@@ -124,35 +126,78 @@ class MemberRemoteDataSourceImpl private constructor(private val memberApi: Memb
         })
     }
 
-    override fun checkEmail(email: String, callBack: MemberCallBack<Boolean>) {
-        memberService.checkEmail(email).enqueue(object : Callback<ResponseBody> {
-            override fun onFailure(call: Call<ResponseBody>, t: Throwable) {
+    override fun checkEmail(email: String, callBack: MemberCallBack<EmailResponse>) {
+        memberService.checkEmail(email).enqueue(object : Callback<EmailResponse> {
+            override fun onFailure(call: Call<EmailResponse>, t: Throwable) {
                 Log.e("tag", t.toString())
             }
 
-            override fun onResponse(call: Call<ResponseBody>, response: Response<ResponseBody>) {
+            override fun onResponse(call: Call<EmailResponse>, response: Response<EmailResponse>) {
                 if (response.code() == SUCCESS) {
-                    callBack.onSuccess(true)
-                } else if (response.code() == CRASH_ERR) {
-                    callBack.onSuccess(false)
+                    response.body()?.let { callBack.onSuccess(it) }
                 }
-                Log.d("checkEmail", response.code().toString())
             }
 
         })
     }
 
-    override fun checkNickname(nickname: String, callBack: MemberCallBack<Boolean>) {
-        memberService.checkNickname(nickname).enqueue(object : Callback<ResponseBody> {
-            override fun onFailure(call: Call<ResponseBody>, t: Throwable) {
+    override fun checkNickname(nickname: String, callBack: MemberCallBack<NicknameResponse>) {
+        memberService.checkNickname(nickname).enqueue(object : Callback<NicknameResponse> {
+            override fun onFailure(call: Call<NicknameResponse>, t: Throwable) {
                 Log.e("tag", t.toString())
             }
 
-            override fun onResponse(call: Call<ResponseBody>, response: Response<ResponseBody>) {
+            override fun onResponse(call: Call<NicknameResponse>, response: Response<NicknameResponse>) {
+                if (response.code() == SUCCESS) {
+                    response.body()?.let { callBack.onSuccess(it) }
+                }
+            }
+
+        })
+    }
+
+    override fun findEmail(name: String, phone: String, callBack: MemberCallBack<List<EmailResponse>>) {
+        val jsonObject = JsonObject().apply {
+            addProperty("name", name)
+            addProperty("phoneNumber", phone)
+        }
+        memberService.findEmail(jsonObject).enqueue(object : Callback<List<EmailResponse>> {
```

Comment:
```suggestion
        memberService.findEmail(jsonObject)
                .enqueue(object : Callback<List<EmailResponse>> {
```
method chaining convention

Reply:
수정했습니당